src/language/index.tsx

```diff
@@ -8,9 +8,6 @@ import { AsyncStorage } from '@gooddollar/web3sdk-v2'
 export const locales = ['de', 'en', 'es-AR', 'es', 'it', 'he', 'ro', 'ru', 'vi', 'zh-CN', 'zh-TW', 'ko', 'ja']
```

Comment:
why do we have russian locale ?

Reply:
I remember that GoodDollar supports people not governments but it's not the case
if you would look any interview with the russian citizens - they totally supports shelling of the cities and civil infrastructure, they wants more shelling, they wants all cities to be totally destroyed.

just go to YouTube and search a bit. if you didn't knew you'll be shocked. that's a totally mind-washed nation now, unfortunately.

Reply:
done